salt/metalk8s/repo/installed.sls

```diff
@@ -44,6 +44,13 @@ Install repositories manifest:
         name: {{ repositories_name }}
         version: {{ repositories_version }}
         products: {{ products }}
+{%- if 'solutions' in pillar.metalk8s %}
+  {%- set solutions = pillar.metalk8s.solutions.deployed %}
```

Comment:
I think you want the `solutions.configured` here.

Also, could have done `{%- set solutions = pillar.metalk8s.solutions.configured | default({}) %}` I think.

Reply:
this to prevent crash when calling locally where we override all the pillar and solutions are not in it ( refer to the bootstrap script L262)